vernac/comTactic.mli

```diff
@@ -0,0 +1,45 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+(** Tactic interpreters have to register they interpretation function *)
+type tactic_expression = {
+  global : bool;
+  ast : Tacexpr.raw_tactic_expr;
+  continuation : unit Proofview.tactic option;
+}
+type tactic_interpreter = string
+
+val register_tactic_interpreter :
+  string -> (tactic_expression -> unit Proofview.tactic) -> unit
+
+(** Entry point for toplevel tactic expression execution *)
+val solve :
+  pstate:Declare.Proof.t ->
+  Goal_select.t ->
+  info:int option ->
+  (tactic_interpreter * tactic_expression) ->
+  with_end_tac:bool ->
+  Declare.Proof.t
+
+type parallel_solver =
+  pstate:Declare.Proof.t ->
+  info:int option ->
+  (tactic_interpreter * tactic_expression) ->
+  solve:bool ->
+  abstract:bool ->
+  with_end_tac:bool ->
+  Declare.Proof.t
```

Comment:
Please document `parallel_solver` and `solve`

Reply:
done

Reply:
thanks!